st2tests/integration/mistral/test_wiring.py

```diff
@@ -39,7 +39,7 @@ def _execute_workflow(self, action, parameters):
         self.assertIn(execution.status, ['scheduled', 'running'])
         return execution
 
-    def _wait_for_completion(self, execution, wait=10):
+    def _wait_for_completion(self, execution, wait=20):
```

Comment:
With 10, the tests were failing intermittently. SQLAlchemy needs things to be tweaked.
See https://gist.github.com/lakshmi-kannan/412bde4abe1233b8433f